### PR TITLE
DBZ-3117 Config validation for Vitess

### PR DESCRIPTION
**Pull request by keweishang**
https://issues.redhat.com/browse/DBZ-3117

### CODE REVIEW

File: src/main/java/io/debezium/connector/vitess/VitessConnector.java [hunk 1]
```diff
@@ -54,4 +60,29 @@ public ConfigDef config() {
...
+                connection.execute("SHOW DATABASES");
+                LOGGER.info("Successfully tested connection for {} with user '{}'", connection.connectionString(), connection.username());
+            }
+            catch (StatusRuntimeException e) {
+                LOGGER.info("Failed testing connection for {} with user '{}'", connection.connectionString(), connection.username());
+                hostnameValue.addErrorMessage("Unable to connect: " + e.getMessage());
+            }
+        }
+        catch (Exception e) {
+            LOGGER.error("Unexpected error shutting down the database connection", e);
```
Comment (gunnarmorling) [inline src/main/java/io/debezium/connector/vitess/VitessConnector.java:80]:
```suggestion
            LOGGER.error("Unexpected error validating the database connection", e);
```

I think that's what it should say? And an error message should be returned to the validation consumer?

**Review by gunnarmorling — COMMENTED** [review]
One comment inline.

**Review by rgibaiev — APPROVED** [review]
Agree with Gunnar's comment

File: src/main/java/io/debezium/connector/vitess/VitessConnector.java (hunk 1 above)
Comment (keweishang) [reply src/main/java/io/debezium/connector/vitess/VitessConnector.java]: Good catch! Addressed

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
LGTM, merged. Thx!

**Pull request merged**